plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeAnalyzeProperties.java

```diff
@@ -39,6 +40,7 @@
 {
     public static final String FILES_MODIFIED_AFTER = "files_modified_after";
     public static final String COLUMNS_PROPERTY = "columns";
+    public static final String FORCE_RECALCULATE_STATISTICS = "force_recalculate_statistics";
```

Comment:
What about using `full_refresh` ?

Reply:
wouldn't it have the same problem as you mentioned with `recalculate_all_stats` ?

Reply:
ok, no problem

Reply:
ok let's go with full_refresh